server/sonar-server/src/main/java/org/sonar/server/rule/RegisterRules.java

```diff
@@ -60,6 +59,8 @@
 
 import static com.google.common.base.Preconditions.checkNotNull;
 import static com.google.common.collect.Lists.newArrayList;
+import static com.google.common.collect.Sets.difference;
```

Comment:
But there'll be some refactoring of these 2 classes as some optimization needs to be done, so it will be a good opportunity to improve the name !